packages/hub/config/structure.sql

```diff
@@ -1095,6 +1062,26 @@ CREATE TABLE public.prepaid_card_patterns (
 
 ALTER TABLE public.prepaid_card_patterns OWNER TO postgres;
 
+--
+-- Name: profiles; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.profiles (
+    id uuid NOT NULL,
+    name text NOT NULL,
+    slug text NOT NULL,
```

Comment:
We're querying by slug so would be good to add an index

Reply:
eep ya, I missed [that](https://github.com/cardstack/cardstack/pull/3120/files#diff-e03f308a75e917163fbf9fe5bc34d68936b918230eea5447cb6f580bf7af8c70L1484), thanks! Included in 1361d9d